wandb/sdk_py27/internal/sender.py

```diff
@@ -305,6 +326,9 @@ def send_request_defer(self, data):
                 # TODO(jhr): now is a good time to output pending output lines
                 self._fs.finish(self._exit_code)
                 self._fs = None
+        elif state == defer.FLUSH_DEBOUNCER:
```

Comment:
FLUSH_DEBOUNCER should definitely be defined to run before FLUSH_FS
since it would be nice that if we send exit code we are done with the run. 
for example state=finished should imply that the config is actually up-to-date

And I would actually run it before FLUSH_DIR.   So we can use that to pick up file changes that came from debounced writes.

Reply:
done